reference/clever-tools/networkgroups.md

```diff
@@ -0,0 +1,192 @@
+---
+title: How to use `clever networkgroups`
+shortdesc: Configure networkgroups using the Clever Cloud CLI tool
+tags:
+- cli
+- reference
+- networkgroups
+keywords:
+- cli
+- clever-tools
+- join
+- wireguard
+- network
+- vpn
+- client
+- server
+---
+
+## Overview
+
+> TODO: Describe networkgroups
+
+## How to create & join networkgroups with the `clever` CLI?
+
+Under each command, I added an example output for you to see what you're supposed to get.
+
+> **Disclaimer:** It might not be up-to-date. Please warn us or <TODO: explain how to contribute on GitHub> if you see anything that's changed.
+
+**General tips:**
+
+- `clever ng` is an alias for `clever networkgroups`. I'll use it to make commands more readable.
+- Command arguments order do not matter, but to have similar commands, I used the convention of placing `--ng NG_ID_OR_LABEL` just after `clever ng`.
```

Comment:
Please remove all use of the first person. 
"It makes commands more readable", 

As a rule of thumb, the first person is best replaced by using the passive voice. Instead of
"I used the convention of placing `foo` just after `bar`
write:
"the convention of placing `foo` just after `bar` is preferred".